src/com/amazon/corretto/crypto/provider/AesGcmSpi.java

```diff
@@ -971,17 +955,22 @@ protected int engineDoFinal(final ByteBuffer input, final ByteBuffer output)
     return output.position() - initialPosition;
   }
 
-  private void checkOutputBuffer(final int inputLength, final byte[] output, final int outputOffset)
```

Comment:
These changes are all covered in the other PR. I couldn't figure out if Github supports commit slicing for PR.

https://github.com/corretto/amazon-corretto-crypto-provider/pull/297